model/domain/tag.go

```diff
@@ -6,22 +6,25 @@ import (
 )
 
 type Tag struct {
-	Key   string
-	Value string
+	Key        string
```

Comment:
might be it makes sense to change 'Key' field type to byte array as well?

Reply:
I can imagine putting binary data in values (like encryption keys or another protobuf structure), but less so in the key. It can still be upgraded though (string and bytes being wire compatible).

Reply:
For example Redis support any type of key, just has some limits of length and recommendations, but for me any type of key is acceptableю

Reply:
I made `Tag.Key []byte` as well.